autoload/multiple_cursors.vim

```diff
@@ -1239,7 +1236,9 @@ function! s:wait_for_user_input(mode)
         exec 'let s:char = "'.substitute(char_mapping, '<', '\\<', 'g').'"'
         break
       endif
-      sleep 1m
```

Comment:
we're not sleeping in this loop anymore... will that cause the CPU to spin for a full 250ms?

Reply:
Thanks for the catch!
The reason why it got removed in the first place was because it can
produce misleading timings (especially on slow system).
The workaround is to sleep for long enough (100ms) at each call to
avoid sleep's call overhead impacting the total time too much.